.github/files/e2e-tests/map-plugins-for-e2e-env.sh

```diff
@@ -0,0 +1,22 @@
+if [[ -z "$BUILD_DIR" ]]; then
+	echo "::error::BUILD_DIR must be set"
+	exit 1
+fi
+
+if [[ -z "$PROJECT_PATH" ]]; then
+	echo "::error::PROJECT_PATH must be set"
+	exit 1
+fi
+
+tar --xz -xvvf "$BUILD_DIR/jetpack-build/build.tar.xz" -C "$BUILD_DIR"
+
+SLUG=$(jq -r -e ".ci.pluginSlug" "$PROJECT_PATH/package.json")
+MIRROR=$(jq -r -e ".ci.mirrorName" "$PROJECT_PATH/package.json")
+
+echo "$MIRROR"
```

Comment:
Looks like a debugging leftover. Do we need this?

Reply:
Leftover indeed. It's not needed.

Reply:
Removed with 616a661